indico/modules/events/management/templates/_settings.html

```diff
@@ -220,7 +220,7 @@
                 </dd>
             {% endif %}
             {% if has_event_labels %}
-                <dt>{% trans %}Label{% endtrans %}</dt>
+                <dt>{% trans "Classification" %}Label{% endtrans %}</dt>
```

Comment:
```suggestion
                <dt>{% trans 'Classification' %}Label{% endtrans %}</dt>
```

Single quotes are superior! 😛